serverless.js

```diff
@@ -91,9 +91,6 @@ module.exports = {
     'serverless-offline': {
       useChildProcesses: true, // needed for hot reloading to work https://github.com/dherault/serverless-offline/issues/931
     },
-    dotenv: {
-      path: 'apps/asap-server/.env',
```

Comment:
There is also a 'apps/asap-server/.env.example'

Reply:
Where are we loading the .env file? the one in the root directory is aimed at Docker and public. Could probably load a .env.local but it likely pickup all the docker variables and vice versa

Reply:
Oh, I thought I had moved the `.env.example` to the root

Reply:
Oh, the existing `.env` is a problem. `dotenv` recommends against having multiple `.env` files, and against committing `.env`. I don't really have an understanding of the backend configuration, @fampinheiro @johnytiago can you advise how we could eliminate the existing `/.env`? Can they just be defaults somewhere in code?

Reply:
I've solved it, hopefully. Can you take another look?